basm.go

```diff
@@ -21,6 +21,14 @@ func Log(msg string) {
 	hostFuncBufferLog(msg)
 }
 
+// LogToHost writes a message to standard error of the host. Because it logs to
```

Comment:
Please take a look at this doc string.  I found it a little tricky to write because I do want to explain why someone would use it while not getting into too much detail.